WordPress/jetpack_metadata/PlayStoreStrings.po

```diff
@@ -68,7 +67,7 @@ msgstr ""
 
 #. translators: Title to be displayed in the Play Store. Limit to 30 characters including spaces and commas!
 msgctxt "app-store-name"
-msgid "Jetpack: WP Security & Speed"
+msgid "Jetpack"
```

Comment:
See pxLjZ-6Vi-p2#comment-18896

Note: the corresponding `jetpack_metadata/title.txt` file was updated in 26548d831405f663ba07187a466640d2cba8adf1 which already got merged to trunk in #16139